bodhi/client/__init__.py

```diff
@@ -298,6 +298,8 @@ def overrides():
 @overrides.command('query')
 @click.option('--user', default=None,
               help='Updates submitted by a specific user')
+@click.option('--mine', 'user', flag_value=os.getenv('USERNAME'),
+              help='Updates submitted by current user')
```

Comment:
I think we should use ```is_flag=True``` on this with ```default=False```. It seems like it isn't very useful to pass an argument to ```--mine``` since that can be done with ```--user``` already.

Also, we should add a callback/validator since it would be an error to use ```--mine``` without setting the ```USERNAME``` environment variable for now. The validator could basically just say "You must set the USERNAME environment variable to use --mine" and raise the appropriate click Exception. For an example of a validator, see ```_warn_if_url_and_staging_set()``` in this same file.

We do plan to rework how authentication works in the Bodhi CLI sometime soon, and I'm hoping we will be able to know the user's name without environment varaibles or CLI flags, but for now I think this is an OK way to do this.